scripts/local/.env

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+# Required Environment Variables for Local setup
+# -----------------------------------------------------------------------------
+#
+# Please locate and paste your string values here to set up 
+# 
+# DO NOT add quotation marks to values
```

Comment:
could you make this a list of actionable items? e.g. 1.... 2..... 3...

Reply:
done